infrastructure/containers/postgres/automatic-backup/scripts/wait-for-postgres-init.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# wait-for-postgres-init.sh
+
+# Checks the container logs until the database has been fully initialize
+until psql -lw -U $POSTGRES_USER --host $POSTGRES_HOST -d $POSTGRES_DB | grep $POSTGRES_DB; do
+  >&2 echo "Postgres initialization in progress - sleeping"
+  sleep 1
+done
+
+sleep 3
+
+>&2 echo "Postgres is initialized - Ready to restore"
+exec "$@"
```

Comment:
What is this line? Is it running the actual docker command passed to the container?

Reply:
The sleep just provides a buffer, the echo is just a normal print statement, and the exec "$@" might actually be unneccesary because running this script is the only command/argument in the command tthat runs it

Reply:
From what I found exec $@ will execute scripts/startup-script.sh since that's what's passed into the entrypoint, pretty neat.

Reply:
testing this right now

Reply:
More specifically, can you add a comment explaining it? Also the `>&2` redirects. Are those redirecting the `echo` messages somewhere?

Reply:
just got rid of them. theyre unnecessary

Reply:
Agreed, this shorthand stuff in bash is weird. PowerShell has similar that can be confusing like $_.